keps/sig-multicluster/cluster-inventory/README.md

```diff
@@ -294,6 +294,11 @@ the API proposed by this KEP aims to
   cluster inventory API for the purpose of workload distribution, operation
   management etc. The consumer SHOULD not create this API resource.
 
+- **Member Cluster**: A kubernetes cluster that is managed by the cluster
+manager. A cluster manager SHOULD have sufficient permission to access
+the member cluster to fetch the information so it can update the status
+of the cluster inventory API resource.
```

Comment:
Awesome.

Only one `spoke cluster` left in the description of `Cluster Manager` on above.
https://github.com/qiujian16/k8s-enhancements/pull/4/files#diff-79aeff7187b5ffc40b00fb82b65aba43131074801edc7e26283a5f833a226420R286-R287